client/my-sites/checkout/concierge-session-nudge/style.scss

```diff
@@ -1,3 +1,85 @@
+.concierge-session-nudge {
+	b {
+		font-weight: 700;
+	}
+}
+.concierge-session-nudge__column-pane {
+	display: flex;
+	flex-flow: row;
+}
+
+.concierge-session-nudge__column-content {
+	padding-right: 10px;
+	@include breakpoint( '<960px' ) {
+		padding-right: 0;
+	}
+}
+
+.concierge-session-nudge__column-doodle {
+	width: 150px;
+	padding-left: 10px;
+	@include breakpoint( '<960px' ) {
+		display: none;
+	}
+}
+
+.concierge-session-nudge__doodle {
+	width: auto;
+	margin-top: 85px;
+}
+
+.concierge-session-nudge__header {
+	text-align: center;
+	font-weight: bold;
+}
+
+.concierge-session-nudge__sub-header {
+	font-size: 21px;
+	font-weight: normal;
+	line-height: 1.4;
+	margin-top: 5px;
+	margin-bottom: 0.75em;
+
+	@include breakpoint( '<1280px' ) {
+		font-size: 21px;
+	}
+
+	@include breakpoint( '<660px' ) {
+		font-size: 18px;
+	}
+}
+
+.concierge-session-nudge__checklist {
+	display: block;
+	width: 100%;
+	text-align: left;
+	margin: 0 0 0.75em;
+	padding: 0;
```

Comment:
If I'm not missing anything, the line 53, 54, 55, 57, 58, and 63 don't seem to be necessary.